docs/reference/ml/anomaly-detection/ml-delayed-data-detection.asciidoc

```diff
@@ -2,8 +2,9 @@
 [[ml-delayed-data-detection]]
 = Handling delayed data
 
-Delayed data are documents that are indexed late. That is to say, it is data
-related to a time that the {dfeed} has already processed.
+Delayed data are documents that are indexed late. That is to say, it is data 
+related to a time that your {dfeed} has already processed and it is therefore
+not analyzed by your {anomaly-job}.
```

Comment:
```suggestion
never analyzed by your {anomaly-job}.
```

IDK which is better. `never` or `not` provide similar meanings. `never` feels more definite